ydb/library/yql/udfs/common/string/string_udf.cpp

```diff
@@ -53,17 +53,37 @@ namespace {
 
 
 // 'unsafe' udf is actually strict - it returns null on any exception
-#define STRING_UNSAFE_UDF(udfName, function)                           \
-    SIMPLE_STRICT_UDF(T##udfName, TOptional<char*>(TOptional<char*>)) {\
-        EMPTY_RESULT_ON_EMPTY_ARG(0);                                  \
-        const TString input(args[0].AsStringRef());                    \
-        try {                                                          \
-            const auto& result = function(input);                      \
-            return valueBuilder->NewString(result);                    \
-        } catch (yexception&) {                                        \
-            return TUnboxedValue();                                    \
-        }                                                              \
-    }
+#define STRING_UNSAFE_UDF(udfName, function)                                        \
+    BEGIN_SIMPLE_STRICT_ARROW_UDF(T##udfName, TOptional<char*>(TOptional<char*>)) { \
+        EMPTY_RESULT_ON_EMPTY_ARG(0);                                               \
+        const TString input(args[0].AsStringRef());                                 \
+        try {                                                                       \
+            const auto& result = function(input);                                   \
+            return valueBuilder->NewString(result);                                 \
+        } catch (yexception&) {                                                     \
+            return TUnboxedValue();                                                 \
+        }                                                                           \
+    }                                                                               \
+                                                                                    \
+    struct T##udfName##KernelExec                                                   \
+        : public TUnaryKernelExec<T##udfName##KernelExec>                           \
+    {                                                                               \
+        template <typename TSink>                                                   \
+        static void Process(TBlockItem arg1, const TSink& sink) {                   \
+            if (!arg1)                                                              \
```

Comment:
style: pls use {} (can be fixed in next PR)

Reply:
Fixed, thanks!